### ISSUE TITLE
notekami; snippet not being populated

### ISSUE THREAD

**Issue by no1etal** [gotbletu/fzf-nova#2]
**Description:**
- **What Happened:** When I open snippet option, there is nothing there. 
- **Expected Behavior:** All other options work as intended
- **Steps to Reproduce:**
  1. Create cheatsheet folder in .config/notekami/foo/example.sh
  2. C-Space + Tab
  3. Notekami -> snippet
  4. Not populating
- **Environment:**
  - Arch
  - Terminal: Kitty and Tmux- **Additional Context:**

**Comment by gotbletu** [comment]
it should be in ~/.config/notekami/cheatsheet/ it will scan anything in there recursively for snippets

~/.config/notekami/cheatsheet/example.sh
~/.config/notekami/cheatsheet/cpp/cout.cpp
~/.config/notekami/cheatsheet/bash/printf.sh

**Comment by no1etal** [comment]
I followed exact structure of files and dirs and also different one to no avail. 
`ls ~/.config/notekami/cheatsheet/`
`bash  cpp  example.sh  shellcmd`

I have read permissions too.

Scratchpaper works fine and saves new .txt files as expected.

- Tried running it as a script and it is the same problem: everything works except snippets.
- Tried downloading _notekami from your git, maing it executable and running it. Issue remains.

I have checked dependencies and `renameutils is NOT installed` because of conflict with `imv` package.
